packages/teams-js/src/internal/telemetry.ts

```diff
@@ -70,6 +70,8 @@ export enum ApiName {
   Call_StartCall = 'call.startCall',
   Clipboard_Read = 'clipboard.read',
   Clipboard_Write = 'clipboard.write',
+  Chat_OpenChat = 'chat.openChat',
+  Chat_OpenGroupChat = 'chat.openGroupChat',
   Dialog_AdaptiveCard_Bot_Open = 'dialog.adaptiveCard.bot.open',
```

Comment:
these two lines go above Clipboard lines?

Reply:
Oh good catch. I would write it down and adjust in 4th batch. Thank you.

Reply:
updated